### ISSUE TITLE
Include I2P

### ISSUE THREAD

**Issue by parazyd** [headslive/bugtracker#1]
```
crosspost from old git.devuan.org repo
http://archive.is/hPkYY
```

Hello! I realize this project is very new, but it seems promising since it is libre and soon-to-be better version of Tails. Thanks for the work you are doing!

Unfortunately, I2P was removed from Tails - since they lost their package maintainer. If I2P could be included in heads that would be awesome! I2P-Bote is a much more secure method of e-mail when combined with GPG.

Notes: http://news.softpedia.com/news/tails-2-11-anonymous-live-cd-is-the-last-to-support-the-i2p-anonymizing-network-513660.shtml https://tails.boum.org/contribute/design/I2P/

-----------------------------------------

Also Privoxy can be used combine TOR + I2P at the same time:

/etc/privoxy/config:
```
forward-socks5   /               127.0.0.1:9050 .
        forward   .i2p               127.0.0.1:4444
```

**Comment by parazyd** [comment]
Yes, this is under consideration for further down the road. Thanks!

**Comment by cfcs** [comment]
If you include i2p, please make the decision to start it an optional thing.

Re: i2p-bote: better than email, probably quite a lot worse than https://github.com/agl/pond - not nearly the same anonymity properties, and lacks forward secrecy, for example.

Re: Privoxy: If you enable that config, websites will be able to tag your traffic, making it easier to identify you. Ideally you would have two different browsers for the two different networks.

**Comment by sigoa** [comment]
in the meantime , just use bitmessage

works OK until MEM leak shuts down the machine
